<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1045</width>
    <height>770</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MIL-STD-1553B Message Processor v1.00.00</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>60</y>
      <width>1021</width>
      <height>691</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>75</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="messageProcessorCheckBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>MejssageProcessorCheckBox</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="configurationCheckBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>161</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>ConfigurationCheckBox</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="qssEditorCheckBox">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>181</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>QSSEditorCheckBox</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="vlcCheckBox">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>VLCCheckBox</string>
    </property>
   </widget>
   <zorder>pushButton</zorder>
   <zorder>messageProcessorCheckBox</zorder>
   <zorder>tabWidget</zorder>
   <zorder>configurationCheckBox</zorder>
   <zorder>qssEditorCheckBox</zorder>
   <zorder>vlcCheckBox</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
